src/core/Form/Textarea/Textarea.test.tsx

```diff
@@ -288,4 +288,38 @@ describe('props', () => {
       expect(container.firstChild).toHaveAttribute('style', 'width: 100px;');
     });
   });
+
```

Comment:
Could also add tests for new props: ariaCharactersRemainingText, ariaCharactersExceededText